Tella/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -264,13 +273,33 @@ If you want to take a picture or a video, proceed in the following way:\
 "Setting_Server_Uwazi_Connect_Server" = "Connected to server";
 "Setting_Server_Uwazi_Success_Message" = "You have successfully connected to the server and will be able to share your data.";
 
+"Setting_Server_GDrive_Select_Type_Toolbar" = "Select Google drive";
```

Comment:
What do you think about updating the keys to this :

"Setting_Server_GDrive_SelectType_AppBar" = "Select Google drive";
"Setting_Server_GDrive_SelectType_Subhead" = "Select a Drive to connect to";
"Setting_Server_GDrive_SelectType_Expl" = "You can either connect to an organizational Shared Drive or create a new folder in your personal Drive.";
"Setting_Server_GDrive_SelectType_Shared" = "USE SHARED DRIVE";
"Setting_Server_GDrive_SelectType_Personal" = "USE PERSONAL DRIVE";
"Setting_Server_GDrive_SelectType_MoreInfo" = "Learn more about the types of drives";
"Setting_Server_GDrive_SelectShared_AppBar" = "Select shared drive";
"Setting_Server_GDrive_CreatePersonal_Subhead" = "Create new folder";
"Setting_Server_GDrive_CreatePersonal_Expl" = "Your reports will be uploaded to a new folder on your Google Drive. Choose a name for this folder here.";
"Setting_Server_GDrive_CreatePersonal_Placeholder" = "Folder name";